spec/src/antlr/JDQL.g4

```diff
@@ -0,0 +1,92 @@
+grammar JDQL;
+
+statement : select_statement | update_statement | delete_statement;
+
+select_statement : select_clause? from_clause? where_clause? orderby_clause?;
```

Comment:
Let's talk about this on the call today. 

But to summarize my view on this, it seems to me that, given:

```java
@Query("where title like '%Data%'")
List<Book> dataBooks();
```

Then it's quite trivial for the provider to transform this query to: `select b from Book as b where b.title like '%Data%'` and that this transformation is completely unambiguous[1].

And my perception is that to ask the user to write:

```java
@Query("select b from Book as b where b.title like '%Data%'")
List<Book> dataBooks();
```

is just contrary to what I perceive to be the goals of Jakarta Data. That's three times as many tokens!

[1] modulo a teensy issue with the syntax for enum literals, but that's surely solvable!

Reply:
```java
@Repository
interface Garage extends BasicRepository<Car, Long>{
@Query("where title like '%Data%'")//it will inject the selct * from Entity as syntax sugar
List<Car> cards();
}
```